### PR TITLE
Update to allow custom User classpath and also correctly implement Passport 'auth:api' guard

### PR DESCRIPTION
**Pull request by b8ne**
As the title suggests and as mentioned in #3 
Note that although this is working for my case, given no tests exist it may fail on edge cases so the main NOTE in README.md has been left.

### CODE REVIEW

**Comment by b8ne** [comment]
@LukeTowers I've been using this in production since this PR with no problems.  Let me know if it needs anything extra in terms of testing or migration.  I think it definitely adds to the usability of the platform and is a great point of difference for Winter to be developer-friendly.

**Comment by LukeTowers** [comment]
Have you tested it with v1.1 (ie Laravel 6)?

File: models/settings/fields.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+fields:
+    user_class:
+        label: "User Model Class"
+        span: auto
+        required: 1
+        type: text
+        comment: "i.e. Backend\\Models\\User. Default: LukeTowers\\Passport\\Models\\BackendUser"
```
Comment (LukeTowers) [inline models/settings/fields.yaml:7]: @b8ne if you're still interested in getting this merged in as a Winter plugin, then it would probably be better to handle this configuration in a config file rather than a DB based settings item. If not, just let me know and I'll make the necessary tweaks and transfer it over to the winter namespace.

Comment (b8ne) [reply models/settings/fields.yaml:7]: It actually is in a config file, see line 77 of config/config.php.  I just use the config, but thought other people may find the GUI way easier.  If we scrap the settings and just use the config, then the entire models/Settings.php implementation can also be removd.

Comment (b8ne) [reply models/settings/fields.yaml:7]: > Have you tested it with v1.1 (ie Laravel 6)?

Just updated dependencies and tested on Laravel 6.
